src/styles/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  margin: 0;
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+  box-sizing: border-box;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-inline: 19px;
+  }
+
+  &__title {
+    margin: 40px $margin-card 0;
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 4px 84px 0 $margin-card;
```

Comment:
Consider using padding for the whole card to limit the text content. The text here may change and the big margin could break the layout.